README.md

```
@@ -254,6 +256,12 @@ Alternatively, you may set the environment variable `GLOBALPING_TOKEN`, which wi
 
 After learning the basics, you may also be interested in these extra features, which provide additional control over your measurements.
 
+#### Set a test timeout
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)